### ISSUE TITLE
[Bug]: Focus indicator border should have interior 1px border

### ISSUE THREAD

**Issue by yvonne-chien-ms** [microsoft/fluentui#24092]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/concepts-developer-accessibility-focus-indicator--page
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/concepts-developer-accessibility-focus-indicator--page

## Actual Behavior
`createFocusOutlineStyle` uses a 2px black border (white in dark mode). 

## Expected Behavior
`createFocusOutlineStyle` should have a 1px white border on the inside of the 2px black border (flipped in dark mode). This will help focus indicator pass contrast requirements when an element has less contrast against the 2px border.

<img width="1800" alt="image" src="https://user-images.githubusercontent.com/91216814/181075901-e76ffb52-be9d-4382-a293-3a0599e6ab0d.png">

Reference: https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/.Design-Tokens-Superset?node-id=4576%3A22603

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by JustSlone** [comment]
Assigned to @microsoft/teams-prg as current owner of Focus, if we want to hand off Focus to another crew we can. High priority because accessibility issue and might be breaking to products. We should consider whether the risk of break is worth the change at this time.
